ser/inference.py

```diff
@@ -0,0 +1,63 @@
+# inference.py
+
+import torch
+from pathlib import Path
+from ser.data import test_dataloader
+from ser.transforms import transforms, normalize
+from ser.params import Params
+import json
+
+def run_infer(run_path: str):
+    label = 6
+
```

Comment:
Ideally the label is not hardcoded but something passed in by the user. Maybe they want an example of an image of number 3 or 8 etc.